### PR TITLE
Escape quote marks fixed

### PR DESCRIPTION
**Pull request by sualfred**
This will fix Hitchers reported issue
https://github.com/phil65/script.toolbox/issues/6

### CODE REVIEW

**Comment by phil65** [comment]
Thx. @HitcherUK, can you confirm?

**Comment by HitcherUK** [comment]
Unfortunately not and there's now and error. See debug log - http://pastebin.com/GU94ZyUz

**Comment by sualfred** [comment]
Strange, for me it works well:
http://i.imgur.com/nDjChdjl.png
http://imgur.com/zR5dQn8

**Comment by HitcherUK** [comment]
What are you wrapping them in?

**Comment by sualfred** [comment]
<onclick>RunScript(script.toolbox,info=textviewer,header=$INFO[ListItem.Label],text='"$VAR[Plot]"')</onclick>

It's implemented the same way you did it.
I tried it with all of my dummy files here and don't have any issues (Running Jarvis RC1).

**Comment by HitcherUK** [comment]
My mistake, must have cached something as a reboot fixed it but there's still an issue when there's an odd number of quotes in the text.

**Comment by sualfred** [comment]
Can you give me the exact $VAR string of your Aliens plot?

**Comment by HitcherUK** [comment]
It's pulled from here - https://www.themoviedb.org/person/10205-sigourney-weaver

**Comment by sualfred** [comment]
@HitcherUK 
Can you test this please?
https://github.com/sualfred/script.toolbox/commit/6444c8d3f3549144eac418bb893533eb8a75e910

And run the textviewer with double quotes at the end and beginning:
onclick>RunScript(script.toolbox,info=textviewer,header=,text='""$INFO[Window.Property(Biography)])""'</onclick>

**Comment by sualfred** [comment]
Okay, that's work for our Ripley, but not for all items. Damn.

**Comment by HitcherUK** [comment]
Haha, even number of quotes now?

**Comment by HitcherUK** [comment]
Phil was thinking about changing it so we used a window property instead if that helps.

**Comment by sualfred** [comment]
Err.... Try it this way with toolbox version:
ExtendedInfo: text='$ESCINFO[Window.Property(Biography)]'
Common windows: text='$ESCINFO[$VAR[Plot]]'

Tested it with a lot of items and no more issues Oo Isn't ESCINFO the same as wrapping the info label in ""?

**Comment by phil65** [comment]
this here should be much nicer:
https://github.com/phil65/script.toolbox/commit/3f4525e0a1ddc5b3a1fbe6dd654fb8d5246124e5
That way you have to set window properties before doing RunScript(). Should be much less prone to errors.

**Comment by sualfred** [comment]
Won't work in our usecase because of the $VAR's
Please wait for Hitchers feedback of the ESCVAR/ESCINFO thing. I browsed dozens of movies/actors in my local library and in ExtendedInfo without any issues.

**Comment by phil65** [comment]
SetProperty(xxx,$VAR[xxx]) does not work? Not sure atm.

**Comment by phil65** [comment]
...would be quite easy to solve with several condtional onclicks though.

**Comment by sualfred** [comment]
Yep, SetProperty(xxx,$VAR[xxx]) doesn't work. Property will be empty.

**Comment by phil65** [comment]
then several conditinal onclicks should be fine. Passing long texts via script params is really dirty, I´d rather get rid of that.

**Comment by sualfred** [comment]
I understand your point but multiple $INFO or $VAR won't work for properties :)

Example:
<onclick>SetProperty(Textviewer.text,$INFO[ListItem.Tagline]$INFO[ListItem.PlotOutline]$INFO[ListItem.Plot],home)</onclick>

**Comment by phil65** [comment]
hmm ok. Then I will wait for Hitcher´s feedback and revert if necessary.

**Comment by sualfred** [comment]
Yep. 
@HitcherUK 
Already added it to my fTV fork for testing if you don't want to rape your code for the testings.

**Comment by HitcherUK** [comment]
@sualfred @phil65 Everything's working perfectly now. Thanks guys.

**Comment by sualfred** [comment]
*dance*

**Comment by phil65** [comment]
Please rebase.

File: default.py [hunk 1]
```diff
@@ -82,8 +80,8 @@ def StartInfoActions(self, infos, params):
             elif info == 'notification':
                 CreateNotification(params.get("header"), params.get("text"), params.get("icon", xbmcgui.NOTIFICATION_INFO), int(params.get("time", 5000)), params.get("sound", True))
             elif info == 'textviewer':
-                xbmcgui.Dialog().textviewer(heading=params.get("header"),
-                                            text=params.get("text"))
+                xbmcgui.Dialog().textviewer(heading=remove_quotes(params.get("header")),
+                                             text=remove_quotes(params.get("text")))
```
Comment (phil65) [inline default.py]: indentation mistake.

**Comment by sualfred** [comment]
I'll never understand how to correctly merge/rebase ^^

**Comment by sualfred** [comment]
@phil65 
Just a frienldy reminder if you forgot to merge it

**Comment by phil65** [comment]
I surely wont pull 4 commits to change 2 lines of code :D
Either you update your PR or I will add those changes "manually".

**Comment by sualfred** [comment]
Adding them manually would be great :D

**Comment by phil65** [comment]
https://github.com/phil65/script.toolbox/commit/53a1709aa589a1faca3def7e44df1e4c0be81f3f

**Pull request closed without merging**